### PR TITLE
Say "wasm32-wasi" instead of "wasm32-unknown-wasi" in wasi-sdk.

### PR DESCRIPTION
**Pull request by sunfishcode**
Hooray for slightly less clutter.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ builds configured to set the default target and sysroot for convenience.
 
 One could also use a standard Clang 8.0, build a sysroot from the sources
```
Comment (pchickey) [inline README.md]: Wait, this won't work with Clang 8.0, but will it work in 8.0.1?

Comment (sunfishcode) [reply README.md]: No, everything here works with Clang 8.0 too.

Comment (sunfishcode) [reply README.md]: If you're watching the LLVM commits, the only patches I landed for this were the ones adding tests for the "wasm32-wasi" triple. The support in the actual code was already present.

Comment (pchickey) [reply README.md]: Ah! I misunderstood that. Thank you for clarifying.

**Pull request merged**